### DISCUSSION TITLE
Enable the option --trace-python-allocators when running the pytest memray plugin?

### DISCUSSION THREAD

**Discussion by godiedelrio** [bloomberg/pytest-memray#76] [General]
I've read through the [doc](https://bloomberg.github.io/memray/python_allocators.html) about how `pymalloc` affects profiling results.
It seems that when running a test suite using the memray pytest plugin, not all the memory allocations from a test would be reported if the pymalloc allocator finds memory available in the arenas and memory pools that had been previously allocated by the tests in the test suite that ran before. Is this something that might happen? In such a case, would help if the pytest memray plugin could be run with the option `--trace-python-allocators` to always track the calls to the pymalloc memory allocator?

**Comment by pablogsal** [comment]
Let's focus this discussion in the issue you opened: https://github.com/bloomberg/pytest-memray/issues/78
